src/fastga/models/aerodynamics/external/unitary_tests/test_externals.py

```diff
@@ -0,0 +1,41 @@
+"""Test module for aerodynamics externals."""
+#  This file is part of FAST-OAD_CS23 : A framework for rapid Overall Aircraft Design
+#  Copyright (C) 2025  ONERA & ISAE-SUPAERO
+#  FAST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from pathlib import Path
+import ast
+from ..xfoil.xfoil_polar import XfoilPolar
+
+
+def test_interpolation_data_type():
+    resources_dir = Path(__file__).parent.parent / "xfoil" / "resources"
+    csv_files_paths = [str(f) for f in resources_dir.glob("*.csv")]
```

Comment:
Do we really need to test all the files ? 
If yes, you can remove the conversion to a str, the results of `.glob()` should be an iterable.